### PR TITLE
feat: Add movie budget in MovieScreen

### PR DESCRIPTION
**Pull request by sdeparte**

### CODE REVIEW

File: src/commonMain/kotlin/com/wizbii/cinematic/journey/presentation/util/NumberFormat.kt [hunk 1]
```diff
@@ -2,6 +2,20 @@ package com.wizbii.cinematic.journey.presentation.util
 
 import kotlin.math.roundToLong
 
+fun Number.formatHumanReadable(): String {
+    val suffix = arrayOf("", "K", "M", "G", "T", "P", "E", "Z", "Y")
+
+    var suffixIndex = 0
+    var value = toDouble()
+
+    while (value > 1000 && suffixIndex < suffix.size - 1) {
```
Comment (Ribesg) [inline src/commonMain/kotlin/com/wizbii/cinematic/journey/presentation/util/NumberFormat.kt]: `suffix.size - 1` -> `suffix.lastIndex`
We could also use `suffixIndex in suffix.indices`

**Review by Ribesg — CHANGES_REQUESTED** [review]
* Display budget chip in the existing set of chips, add some kind of line break to the row of chips
* Migrate DB

File: src/commonMain/kotlin/com/wizbii/cinematic/journey/presentation/util/NumberFormat.kt (hunk 1 above)
Comment (sdeparte) [reply src/commonMain/kotlin/com/wizbii/cinematic/journey/presentation/util/NumberFormat.kt]: Updated

File: src/commonMain/kotlin/com/wizbii/cinematic/journey/data/source/TmdbApiDataSource.kt [hunk 2]
```diff
@@ -157,6 +157,7 @@ class TmdbApiDataSource {
     @Serializable
     data class TmdbMovieDetails(
         val backdropPath: String?,
+        val budget: Long?,
```
Comment (Ribesg) [inline src/commonMain/kotlin/com/wizbii/cinematic/journey/data/source/TmdbApiDataSource.kt:160]: Budget should probably be an Int

File: src/commonMain/kotlin/com/wizbii/cinematic/journey/presentation/screen/movie/MovieScreen.kt [hunk 3]
```diff
@@ -385,6 +389,7 @@ private fun Poster(
 }
 
 @Composable
+@OptIn(ExperimentalLayoutApi::class)
```
Comment (Ribesg) [inline src/commonMain/kotlin/com/wizbii/cinematic/journey/presentation/screen/movie/MovieScreen.kt]: `OptIn` should be applied closer to the experimental use, probably on `FlowRow`

File: src/commonMain/sqldelight/com/wizbii/cinematic/journey/data/migrations/3.sqm [hunk 4]
```diff
@@ -0,0 +1 @@
+ALTER TABLE tmdbMovieRecord ADD COLUMN budget INTEGER;
```
Comment (Ribesg) [inline src/commonMain/sqldelight/com/wizbii/cinematic/journey/data/migrations/3.sqm]: Version 3 already exists, version 4 should be used instead

**Review by Ribesg — CHANGES_REQUESTED** [review]

File: src/commonMain/kotlin/com/wizbii/cinematic/journey/presentation/screen/movie/MovieScreen.kt (hunk 3 above)
Comment (sdeparte) [reply src/commonMain/kotlin/com/wizbii/cinematic/journey/presentation/screen/movie/MovieScreen.kt]: Updated

File: src/commonMain/kotlin/com/wizbii/cinematic/journey/data/source/TmdbApiDataSource.kt (hunk 2 above)
Comment (sdeparte) [reply src/commonMain/kotlin/com/wizbii/cinematic/journey/data/source/TmdbApiDataSource.kt:160]: Updated

File: src/commonMain/sqldelight/com/wizbii/cinematic/journey/data/migrations/3.sqm (hunk 4 above)
Comment (sdeparte) [reply src/commonMain/sqldelight/com/wizbii/cinematic/journey/data/migrations/3.sqm]: Updated

**Review by Ribesg — APPROVED** [review]

**Pull request merged**